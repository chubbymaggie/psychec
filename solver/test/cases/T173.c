void f()
{
    int* t = v;
    *t = 10;
    double* u = v;
}

void f1()
{
    T1* t1 = v1;
    *t1 = 10;
    double* u1 = v1;
}

void f2()
{
    T2* t2 = v2;
    *t2 = 20;
    U2* u2 = v2;
    *u2 = 3.14;
}

void f3()
{
    U3* u3 = v3;
    *u3 = 3.14;
    T3* t3 = v3;
    *t3 = 30;
}
