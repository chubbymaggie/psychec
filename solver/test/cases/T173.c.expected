/* Forward declarations */

/* Type definitions */
typedef  double U3 ;
typedef  double U2 ;
typedef  int T3 ;
typedef  int T2 ;
typedef  int T1 ;

/* Variables and functions */
 void* v ; 
 void* v1 ; 
 void* v2 ; 
 void* v3 ; 

